address_book_query.go

```diff
@@ -0,0 +1,182 @@
+package hedera
+
+import (
+	"context"
+	"github.com/hashgraph/hedera-protobufs-go/services"
+	"io"
+	"math"
+	"time"
+
+	"github.com/hashgraph/hedera-protobufs-go/mirror"
+	"google.golang.org/grpc/status"
+)
+
+//var rstStream = regexp.MustCompile("(?i)\\brst[^0-9a-zA-Z]stream\\b")
+
+type AddressBookQuery struct {
+	errorHandler      func(stat status.Status)
```

Comment:
I'm not sure if these callbacks are really warranted since this is very different from `TopicMessageQuery` since it won't exist as a background task, and instead we'll listen to the entire stream before returning.

Reply:
Yep, still trying to figure all of this out